plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx

```diff
@@ -54,7 +54,7 @@ export const ScaffolderPage = () => {
   const { data: templates, isValidating, error } = useStaleWhileRevalidate(
     'templates/all',
     async () =>
-      catalogApi.getEntities({ kind: 'Template' }) as Promise<
+      catalogApi.getEntities({ filter: 'kind=Template' }) as Promise<
```

Comment:
Noooooope xP. We want to fix `getEntities` so that it behaves correctly instead

Reply:
HAHA. OK